plugin/src/main/java/io/jenkins/plugins/analysis/warnings/OwaspDependencyCheck.java

```diff
@@ -0,0 +1,34 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisModelParser;
+
+/**
+ * Provides parser for OWASP dependency check reports.
+ */
+public class OwaspDependencyCheck extends AnalysisModelParser {
+    private static final long serialVersionUID = -7721519880683487886L;
+
+    private static final String ID = "owasp-dependency-check";
+
+    /** Create instance. */
+    @DataBoundConstructor
+    public OwaspDependencyCheck() {
+        super();
+        // empty constructor required for stapler
+    }
+
+    /** Descriptor for this static analysis tool. */
+    @Symbol("owaspDependencyCheck")
```

Comment:
Did you check if this ID is not already used by the original OwaspDependencyCheck plugin?

Reply:
The original has:

- [`@Symbol("dependency-check")`](https://github.com/jenkinsci/dependency-check-plugin/blob/dependency-check-jenkins-plugin-5.1.1/src/main/java/org/jenkinsci/plugins/DependencyCheck/tools/DependencyCheckInstallation.java#L93)
- [`@Symbol("dependencyCheckPublisher")`](https://github.com/jenkinsci/dependency-check-plugin/blob/dependency-check-jenkins-plugin-5.1.1/src/main/java/org/jenkinsci/plugins/DependencyCheck/DependencyCheckPublisher.java#L173)
- [`@Symbol({"dependencyCheck", "dependencycheck"})`](https://github.com/jenkinsci/dependency-check-plugin/blob/dependency-check-jenkins-plugin-5.1.1/src/main/java/org/jenkinsci/plugins/DependencyCheck/DependencyCheckToolBuilder.java#L220)

[Pipeline Step Reference](https://www.jenkins.io/doc/pipeline/steps/dependency-check-jenkins-plugin/) doesn't list any others for that plugin, either.